mailgreen/app/schemas/subscription.py

```
@@ -0,0 +1,16 @@
+from pydantic import BaseModel
+from uuid import UUID
+from typing import Optional
+
+
+class SubscriptionOut(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)